tool/.babelrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
.DS_Store got committed, I think this is a tmp file on OSx? We should probably remove that and add it to .gitignore

(Comment is here, because I couldn't comment on that file)

Reply:
removed the file and added .gitignore